Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -1041,7 +1060,9 @@ def get_test_list_and_content_packs_to_install(files_string, branch_name, minimu
 
     from_version, to_version = get_from_version_and_to_version_bounderies(all_modified_files_paths, id_set)
 
-    create_filter_envs_file(from_version, to_version)
+    # Check if only README file in file string, if so, no need to create the servers.
+    documentation_changes_only = is_documentation_changes_only(files_string)
+    create_filter_envs_file(from_version, to_version, documentation_changes_only=bool(documentation_changes_only))
```

Comment:
```suggestion
    create_filter_envs_file(from_version, to_version, documentation_changes_only=documentation_changes_only)
```
Why is the casting required?